src/commands/mod.rs

```diff
@@ -268,6 +285,53 @@ where
     map.end()
 }
 
+fn deserialize_string_not_empty<'de, D>(deserializer: D) -> Result<String, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let s: String = Deserialize::deserialize(deserializer)?;
+    if s.is_empty() {
+        return Err(serde::de::Error::custom("String cannot be empty"))
```

Comment:
I asked before - will this show field name ?

Reply:
I've replied back then (https://github.com/Screenly/cli/pull/96#discussion_r1313016497): I don't think that you can get field name here, you get line and column.

Reply:
I see, ok that's fine for now.